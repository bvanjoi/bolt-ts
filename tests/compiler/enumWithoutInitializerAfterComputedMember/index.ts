// From `github.com/microsoft/TypeScript/blob/v5.9.2/tests/cases/compiler/enumWithoutInitializerAfterComputedMember.ts`, Apache-2.0 License

enum E {
    a,
    b = a,
    c
}

enum B {
    a,
    b = '',
    c = b,
    d
    //~^ ERROR: Enum member must have initializer.
}