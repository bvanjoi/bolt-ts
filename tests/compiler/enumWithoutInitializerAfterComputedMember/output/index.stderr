  × Enum member must have initializer.
    ╭─[../../tests/compiler/enumWithoutInitializerAfterComputedMember/index.ts:13:5]
 13 │ d
    · ─
    ╰────
